res/controllers/Reloop-Jockey-3-ME-scripts.js

```diff
@@ -452,9 +446,9 @@ Jockey3ME.MixerVol = function (channel, control, value, status, group) {
     currentDeck += 2;
   }
   if (control == 0x2D || control == 0x6C) {
-	  engine.setValue("[Channel" + currentDeck + "]","pregain",script.absoluteNonLin(value,0,1,4,0,127));
+	  engine.setValue("[Channel" + currentDeck + "]","pregain",script.absoluteLin(value,0,script.absoluteLin(value,0,4,0,127),0,127));
```

Comment:
If this is for Mixxx 2.0, you should consider replacing that call with engine.setParameter() so you don't have to worry about whether the MixxxControl is linear or not. See http://mixxx.org/wiki/doku.php/getting_ready_for_112